cmd/cue/cmd/testdata/script/export_openapi_err.txt

```diff
@@ -0,0 +1,22 @@
+! cue export repeated.cue -o openapi:export.json
+cmp stdout expect-stdout
+cmp stderr expect-stderr
+
+! cue eval repeated.cue --out openapi
+cmp stdout expect-stdout
```

Comment:
you can just do `! stdout .` to verify that stdout is empty; no need for an empty expect-stdout file.